docs/reference/migration/migrate_3_0.asciidoc

```diff
@@ -195,6 +195,22 @@ the parent id under the `_parent` key.
 [[breaking_30_settings_changes]]
 === Settings changes
 
+From Elasticsearch 3.0 on all settings are validated before they are applied. Node level and default index
+level settings are validated on node startup, dynamic cluster and index setting are validated before they are updated/added
+to the cluster state. Every setting must be a _known_ setting or in other words all settings must be registered with the
+node or transport client they are used with. This implies that plugins that define custom settings must register all of their
+settings during pluging loading on the `SettingsModule#registerSettings(Setting)` method.
+
+==== Node settings
+
+The `name` settings has been remove and is replaced with `node.name`. Usage of `-Dname=some_node_name` is not supported
+anymore. Scripts must be updated accordingly.
+
+==== Transport Settings
+
+All settings with a `netty` infix have been replaced by their already existing `transport` synonyms. For instance `transport.netty.bind_host` is
+now not supported anymore and should be replaced by the superseeding setting `transport.bind_host`.
```

Comment:
now not supported anymore -> no longer supported
Delete "the superseeding setting" (or change superseeding -> superseding)